content/soarian/dstu2/general-clinical/procedure.md

```diff
@@ -27,29 +27,25 @@ _Implementation Notes_
 
 * The API returns the most current procedure list.
 
-* The Narrative includes additional information pertaining to the procedures, and should be shown to the user to ensure completeness of clinical content and context.
+* Procedures in erroneous state are not returned.
```

Comment:
Updated the documentation,
Procedures in completed or aborted status are returned.
Procedures in erroneous status are not returned.